db/comdb2.c

```diff
@@ -5715,8 +5715,7 @@ static struct dbenv *newdbenv(char *dbname, char *lrlname)
         return NULL;
     }
 
-    if (lrlname)
-        pre_read_lrl_file(dbenv, lrlname, dbname);
+    if (lrlname) pre_read_lrl_file(dbenv, lrlname, dbname);
```

Comment:
Roborivers uses the coding style in [this doc](https://bloomberg.github.io/comdb2/contrib.html#indentation-etc
).